build.gradle

```diff
@@ -790,6 +791,22 @@ project('quote') {
 	}
 }
 
+project('splunk') {
+	description = 'Twitter Basic Sample'
```

Comment:
Hi @fbalicchia, one more time!

Yeah... This PR need so much redo...
Would you mind to review it yourself first of all and fix all found issues?
Just because I'm going to comment here almost on each line.

https://github.com/spring-projects/spring-integration/blob/master/CONTRIBUTING.md
https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style

should help you to be close to the perfect :smile: 

And it would be grate to make samples based on the Spring Boot! 

From other side you can just let us know how it works for you and we can go ahead with your PR even without comments, but with a lot of rework.